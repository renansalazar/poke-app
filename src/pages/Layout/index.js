import {
  Outlet,
  Link,
} from "react-router-dom";

export default function Layout() {
  return (
    <>
      <header>
        <Link to="/">Pokedex</Link>
      </header>
      <div className='container'>
        <Outlet />
      </div>
    </>
  );
}