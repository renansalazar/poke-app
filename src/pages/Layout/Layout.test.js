import React from 'react';
import {render} from '@testing-library/react';
import  Layout from '.';
import { QueryClient, QueryClientProvider } from 'react-query';
import { BrowserRouter, Routes, Route } from "react-router-dom"

describe('Detalle component', () => {
  test('first call to detail', () => {
    const queryClient = new QueryClient();
    const { container } = render(
      <QueryClientProvider client={queryClient}>
        <BrowserRouter>
          <Layout />
        </BrowserRouter>
      </QueryClientProvider>
    )
    expect(container.firstChild).toMatchSnapshot()
  });
});