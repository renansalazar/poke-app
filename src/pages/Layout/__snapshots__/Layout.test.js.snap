// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detalle component first call to detail 1`] = `
<header>
  <a
    href="/"
  >
    Pokedex
  </a>
</header>
`;
